pkgs/development/python-modules/httmock/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, requests }:
+
+buildPythonPackage rec {
+  pname   = "httmock";
+  version = "1.2.6";
+
+  src = fetchFromGitHub {
```

Comment:
Are you using `fetchFromGitHub` because `tests.py` is not included in the PyPI tarball? I've opened https://github.com/patrys/httmock/pull/57.